src/components/Timeline/TimelineEventHeader.tsx

```diff
@@ -10,68 +13,122 @@ export interface TitleElement {
   link?: string;
 }
 
+export interface TimelineUser {
+  id: string;
+  email: string;
+  firstName?: string;
+  lastName?: string;
+  avatar?: {
+    url?: string;
+  } | null;
```

Comment:
this is ui component, you can simplify props to `avatarUrl?: string`, it doesnt have to map graphql